dev/run-tests-jenkins

```diff
@@ -31,7 +31,7 @@ export LANG="en_US.UTF-8"
 PYTHON_VERSION_CHECK=$(python3 -c 'import sys; print(sys.version_info < (3, 6, 0))')
 if [[ "$PYTHON_VERSION_CHECK" == "True" ]]; then
   echo "Python versions prior to 3.6 are not supported."
-  exit -1
+  exit 255
```

Comment:
SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.